pixart.js

```diff
@@ -1 +1,51 @@
+//initialize color so that by default the brush is same as canvas
+var color = "#E7E5DB";                   // blue: "#1C4273";
 
+var counter = 0;
+
+// on click of "set color" button, change swatch in order and start over width
+// first swatch after cycle through each one. Visible name of swatch should match
+// background color 
+$("button").eq(0).click(function () {
```

Comment:
Since you started with jQuery, I recommend continuing with jQuery (on that same line, especially). `$("button").on("click", ...)`